packages/sds/src/index.ts

```diff
@@ -1,3 +1,5 @@
-import { BloomFilter } from "./bloom.js";
+import { BloomFilter } from "./bloom_filter/bloom.js";
+
+export * from "./message_channel/index.js";
```

Comment:
are all exports needed outside of the package? 
I'd like to be minimalistic in what we export